docker/database/init-db_local.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Please remove this file, since it is not connected with this PR. This should be committed under a new PR.